tests/integration/queries/test_submission.py

```diff
@@ -75,4 +75,6 @@ def test_get_reviews(indico):
     for sub in subs:
         reviews = client.call(GetReviews(sub.id))
         for review in reviews:
-            assert (isinstance(review, SubmissionReviewFull))
\ No newline at end of file
+            assert (isinstance(review, SubmissionReviewFull))
+            if not review.rejected:
+                assert isinstance(review.changes, (list, dict))
```

Comment:
potentially dumb q but why are we not checking if isinstance JSONType? are we okay with this mismatch between expected types versus the type hint

Reply:
the type-hint just forces the type parser to json.load() the object - it's not actually a "JSONType" https://github.com/IndicoDataSolutions/indico-client-python/blob/master/indico/types/base.py#L51

Reply:
i followed this process similar to the Job type class - if there's a different way we want to do this, lmk - but the existing typing was incorrect since `changes` was actually just a json string - not a dict